website/docs/table-group/data-table/data-table-code.md

```diff
@@ -375,45 +402,40 @@ Add a skeleton to the table by directly substituting it in the `data` or replaci
 
 :::
 
-## Columns merging
+## Specific cases
 
-Merge two or more columns by changing the table data and using `/` to combine column keys.
+### Accordion inside table
```

Comment:
maybe create a section for all 3 accordion examples and call it "Accordion"?

Reply:
Not sure, since the last two with tables inside tables look like they are intended to show these specific cases with for nesting tables. But could be a good option.

I took the Accordion section out of the Specific cases and moved examples with tables at the top, so they will follow the Accordion section.

Reply:
I think we can move two examples with table and tanle after we'll investigate a little whether they are necessary at all.